examples/05-Q3D/Q3D_DC_IR.py

```diff
@@ -108,8 +108,9 @@
 # Launch the newly created q3d project and plot it.
 
 q3d = pyaedt.Q3d(output_q3d)
-q3d.plot(show=False, objects=["1_2V_AVDLL_PLL", "1_2V_AVDDL", "1_2V_DVDDL"],
-         export_path=os.path.join(q3d.working_directory, "Q3D.jpg"), plot_air_objects=False)
+
```

Comment:
I would directly remove these lines, because if not they will be created on the website automatically.